flycheck.el

```diff
@@ -9881,6 +9881,10 @@ See URL `http://call-cc.org/'."
             "Warning: " (zero-or-more not-newline) ":\n"
             (one-or-more (any space)) "(" (file-name) ":" line ") " (message)
             line-end)
+   (warning line-start
+            "Warning: " (zero-or-more not-newline) ":\n"
+            (one-or-more (any space)) "(" (file-name) ":" line ") " (message)
+            line-end)
```

Comment:
I might be missing something, but it looks identical to the pattern just above it?

Reply:
Good catch :) I copied to the wrong form from my personal config. I just pushed a new version.